src/com/sun/ts/tests/securityapi/idstore/ldap/notvalidated/ServletForLdapIDStore.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
  *
  * This program and the accompanying materials are made available under the
```

Comment:
The changes on this file do not look needed.

Reply:
Shall be fixed.